core/templates/dev/head/components/forms/schema_editors/schema_based_bool_editor_directive.html

```diff
@@ -4,3 +4,9 @@
            focus-on="<[labelForFocusTarget()]>">
   </span>
 </div>
+<style>
```

Comment:
Again, this file should not be modified at all. Please keep CSS changes scoped to customize_interaction_modal_directive.html. You can use a specific CSS selector (e.g. `customize-interaction-modal .boolean-checkbox`). The `customize-interaction-modal` ensures that only that directive is affected.

Reply:
oh.Okay.got it.Thanks